.eslintrc.json

```diff
@@ -5,17 +5,8 @@
     "commonjs": true,
     "node": true
   },
-  "extends": [
-    "plugin:prettier/recommended",
-    "plugin:@typescript-eslint/recommended",
-    "prettier/@typescript-eslint",
-    "plugin:import/typescript"
-  ],
-  "plugins": [
-    "@typescript-eslint",
-    "prettier",
-    "import"
-  ],
+  "extends": ["plugin:prettier/recommended", "plugin:@typescript-eslint/recommended", "prettier/@typescript-eslint"],
```

Comment:
All those linter settings were really annoying me, so I'm sneaking this change in as well.